modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -2938,7 +2938,9 @@ definition {
 		Workflow.viewDefinitionUnpublished(workflowDefinitionTitle = "New Workflow Definition Title");
 	}
 
+	@description = "Verify that the user can save an invalid workflow definition as a draft"
 	@priority = 3
+	test CanSaveInvalidWorkflowDefinitionAsDraft {
```

Comment:
@davisaints can you check if this test duplicates CanDeleteEndNode or we need to update this? Thanks!

Reply:
Thanks for the review! We can update this test.